blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -92,8 +92,13 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
             final String body = request.getContent().toString(Constants.DEFAULT_CHARSET);
             try {
                 jsonMetricsContainer = createContainer(body, tenantId);
-                if (!jsonMetricsContainer.isValid()) {
-                    throw new IOException("Invalid JSONMetricsContainer");
+
+                List<String> errors = jsonMetricsContainer.getValidationErrors();
```

Comment:
Later at line 121, there seems to be a check for jsonMetricsContainer being null. Do we need to move that check before this line?